code/modules/mob/mob_layering.dm

```diff
@@ -7,22 +7,24 @@
 	var/last_layer = layer
 	var/new_layer = get_base_layer()
 	if(isturf(loc))
-
 		var/turf/T = loc
 		if(T.pixel_z < 0)
 			new_layer = T.layer + 0.25
 		else if(buckled && buckled.buckle_layer_above)
-			new_layer = buckled.layer + ((buckled.dir == SOUTH) ? -0.1 : 0.1)
+			new_layer = buckled.layer + ((buckled.dir == SOUTH) ? -0.01 : 0.01)
```

Comment:
fyi there is a point if layers get too large (this is an issue with wallening on tg, iirc, they multiplied everything by 1000) or if the subdivisions get too small that floating point errors will fuck this up. it's probably not an issue here but like, figured i'd comment on it just in case